create-turbo/src/yarn.ts

```diff
@@ -0,0 +1,10 @@
+import { execSync } from "child_process";
+
+export function checkIfYarnInstalled(): boolean {
```

Comment:
Can we reuse create-turbo/src/shouldUseYarn.ts?

Reply:
@jaredpalmer Ah yes, it's perfect! I didn't look into what the `userAgent` was, and I thought that was checking for something else, learned about a new value today!

Thoughts on renaming `shouldUseYarn`? Reads a bit to me like "are we supposed to use yarn", as opposed to checking if yarn is running/available, which is partially why I didn't look more closely at it in the first place.

Reply:
Keep it for now.